scripts/check_frontend_coverage.py

```diff
@@ -484,7 +484,7 @@ def check_coverage_changes():
                     '\033[1m{}\033[0m seems to be fully covered!'
```

Comment:
@nithusha21, Should we change the filename to `check_frontend_test_coverage`? (`check_frontend_test_coverage` seems more appropriate to me.)